app/src/main/java/org/keynote/godtools/android/fragment/LanguagesFragment.java

```diff
@@ -69,6 +75,9 @@ public void onCreate(@Nullable final Bundle savedInstanceState) {
             mPrimary = args.getBoolean(EXTRA_PRIMARY, mPrimary);
         }
 
+        // Allow Fragment to have its own menu
+        setHasOptionsMenu(true);
```

Comment:
minor nit, my style has usually been to put this call directly under the `super.onCreate()` line.